PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -3522,7 +3551,7 @@ bool ScintillaEditView::markerMarginClick(size_t lineNumber)
 
 		if (openPresent)
 		{
-			_currentBuffer->setHideLineChanged(false, lineNumber);
+			_currentBuffer->setHideLineChanged(false, lineNumber + 1);

```

Comment:
Variable `lineNumber` is always decremented before each iteration in this `for` loop, even before `lineNumber >= 0 && !openPresent` evaluates to false. As a result, after this loop `lineNumber` will always be decremented too much by one. To solve this, one is added back to `lineNumber`